kahuna/public/js/components/gr-icon/gr-icon.css

```diff
@@ -0,0 +1,27 @@
+.gr-icon {
```

Comment:
Could we not just style the `gr-icon` element.... @theefer was using that as a pattern elsewhere and it's quite nice.